organise_desktop/organiseDesktop.py

```diff
@@ -75,6 +75,23 @@ def makedir(self, folders_to_make):
             if not path.isdir(dir):
                 mkdir(dir)
 
+    def removedir(self, folders_i_made):
```

Comment:
Also, this method and undo method, It seems like you copied changes from my PR. but in this way my commits will be buried - If @blavejr accepts both PR, All of the commits will be merged together so This part is not really needed. It will more likely cause a conflict.